cmd/chisel/cmd_info_test.go

```diff
@@ -0,0 +1,250 @@
+package main_test
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	. "gopkg.in/check.v1"
+
+	chisel "github.com/canonical/chisel/cmd/chisel"
+	"github.com/canonical/chisel/internal/testutil"
+)
+
+type infoTest struct {
+	summary string
+	input   map[string]string
+	query   []string
+	err     string
+	stdout  string
+}
+
+var infoTests = []infoTest{{
+	summary: "A single slice inspection",
+	input:   infoRelease,
+	query:   []string{"mypkg_foo"},
+	stdout: `
+package: mypkg
+archive: ubuntu
+slices:
+    foo:
+        contents:
+            /etc/foo: {}
+            /etc/foo-dir/: {make: true, mode: 0644}
+`,
```

Comment:
Let's indent these tests like in `slicer_test.go` to make it feel more natural and then use `testutil.Reindent` when running the test.